### ISSUE TITLE
List index out of range

### ISSUE THREAD

**Issue by gigomen** [donewiththedollar/directional-scalper#11]
![изображение](https://user-images.githubusercontent.com/117885686/227137650-92e756af-dd1a-4748-ae5e-123bedf8304c.png)

Hello, I hope I wrote correctly. the lower part is updated for the first 10 seconds and then errors appear as in the screenshot, but at the same time transactions are opened, I'm not sure if it's correct. Maybe the problem is my location.

**Comment by gigomen** [comment]
I am using the version 1.1.0. It's on the bot.py

**Comment by donewiththedollar** [comment]
Actually after implmenting ``--deleverage`` and ``--avoidfees`` I have also been seeing this message but the bot runs just fine, the orders are set. It seems that perhaps it is due to our position size being too small, so when the calculation is make to break take profit into increments there must be some error. I will look into it :) thank you for bug hunting!! 

But to be clear, I do also see this error but still yielding great results orders are being placed as I would expect even with this error.

**Comment by gigomen** [comment]
Yes, thanks for the help, I have a really small order, I'm testing different ones. But I don’t fully understand how --avoidfees works, but I kind of understand --deleverage. I'm happy to help in any way

**Comment by ecoppen** [comment]
Not clear what this error message relates to in the code, I'll add some extra logging to each exception possibly to narrow it down. Initially looks like api isn't returning data or enough data from a request.

**Comment by gigomen** [comment]
Sorry, the issue is probably resolved, I switched to the cloud

**Comment by ecoppen** [comment]
Great, closing for now - reopen if you come across it again :)
